### PR TITLE
Task/gv 681 loader popup

### PR DESCRIPTION
**Pull request by Anchello**

### CODE REVIEW

File: packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import "../dialog.scss";
+
+import * as React from "react";
+import SvgLoader from "shared/components/svg-loader/svg-loader";
+
+export const DialogLoaderHeader = () => (
```
Comment (pavelzubov) [inline packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx]: Declare type

File: packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  <SvgLoader height={57} width={150}>
+    <rect x="0" y="0" width="150" height="26" rx="8" ry="8" />
+    <rect x="0" y="39" width="100" height="18" rx="8" ry="8" />
+  </SvgLoader>
+);
+
+export const DialogLoaderStat = () => (
```
Comment (pavelzubov) [inline packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx]: Declare type

File: packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  <SvgLoader height={56} width={200}>
+    <rect x="0" y="0" width="200" height="18" rx="8" ry="8" />
+    <rect x="0" y="30" width="180" height="26" rx="8" ry="8" />
+  </SvgLoader>
+);
+
+export const DialogLoaderinput = () => (
```
Comment (pavelzubov) [inline packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx]: Declare type

File: packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  <SvgLoader height={65} width={296}>
+    <rect x="0" y="0" width="100" height="16" rx="8" ry="8" />
+    <rect x="0" y="30" width="296" height="35" rx="8" ry="8" />
+  </SvgLoader>
+);
+
+export const DialogLoaderButton = () => (
```
Comment (pavelzubov) [inline packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx]: Declare type

File: packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  <SvgLoader height={42} width={296}>
+    <rect x="0" y="0" width="296" height="42" rx="26" ry="26" />
+  </SvgLoader>
+);
+
+export const DialogLoaderHeaderGoogleAuth = () => (
```
Comment (pavelzubov) [inline packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx]: Declare type

File: packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  <SvgLoader height={57} width={350}>
+    <rect x="0" y="0" width="350" height="26" rx="8" ry="8" />
+    <rect x="0" y="39" width="100" height="18" rx="8" ry="8" />
+  </SvgLoader>
+);
+
+export const DialogLoaderShortStat = () => (
```
Comment (pavelzubov) [inline packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx]: Declare type

File: packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+  <SvgLoader height={56} width={140}>
+    <rect x="0" y="0" width="50" height="16" rx="8" ry="8" />
+    <rect x="0" y="30" width="140" height="26" rx="8" ry="8" />
+  </SvgLoader>
+);
+
+export const DialogLoaderShortButton = () => (
```
Comment (pavelzubov) [inline packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx]: Declare type

File: packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+  <SvgLoader height={42} width={150}>
+    <rect x="0" y="0" width="150" height="42" rx="26" ry="26" />
+  </SvgLoader>
+);
+
+export const DialogLoaderImage = () => (
```
Comment (pavelzubov) [inline packages/shared/components/dialog/dialog-loader/dialog-loader-elements.tsx]: Declare type

File: packages/shared/components/dialog/dialog-loader/dialog-loader-google-auth.tsx [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+import {
+  DialogLoaderHeaderGoogleAuth,
+  DialogLoaderImage,
+  DialogLoaderShortButton,
+  DialogLoaderShortStat,
+  DialogLoaderinput
+} from "./dialog-loader-elements";
+
+export const DialogLoaderGoogleAuthDesktop = () => (
```
Comment (pavelzubov) [inline packages/shared/components/dialog/dialog-loader/dialog-loader-google-auth.tsx]: Declare type

File: packages/shared/components/dialog/dialog-loader/dialog-loader-google-auth.tsx [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+        <div className="dialog__buttons">
+          <DialogLoaderShortButton />
+        </div>
+      </div>
+    </div>
+  </div>
+);
+
+export const DialogLoaderGoogleAuthMobile = () => (
```
Comment (pavelzubov) [inline packages/shared/components/dialog/dialog-loader/dialog-loader-google-auth.tsx]: Declare type

File: packages/shared/components/dialog/dialog-loader/dialog-loader-google-auth.tsx [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+  <div className="google-auth google-auth--mobile">
+    <DialogLoaderGoogleAuthFirstStep />
+  </div>
+);
+
+const DialogLoaderGoogleAuthFirstStep = () => (
```
Comment (pavelzubov) [inline packages/shared/components/dialog/dialog-loader/dialog-loader-google-auth.tsx]: Declare type

File: packages/shared/components/dialog/dialog-loader/dialog-loader.tsx [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+
+import * as React from "react";
+import {
+  DialogLoaderButton,
+  DialogLoaderHeader,
+  DialogLoaderinput,
+  DialogLoaderStat
+} from "./dialog-loader-elements";
+
+export const DialogLoader = () => (
```
Comment (pavelzubov) [inline packages/shared/components/dialog/dialog-loader/dialog-loader.tsx]: Declare type

File: packages/shared/modules/2fa/google-auth/google-auth-steps/dialog-loader-google-auth-steps.tsx [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+  ui
+}: RootState): IDialogLoaderGoogleAuthStepsStateProps => ({
+  innerWidth: ui.innerWidth
+});
+
+const DialogLoaderGoogleAuthStepsContainer = connect(mapStateToProps)(
+  DialogLoaderGoogleAuthSteps
+);
+
+export default DialogLoaderGoogleAuthStepsContainer;
```
Comment (pavelzubov) [inline packages/shared/modules/2fa/google-auth/google-auth-steps/dialog-loader-google-auth-steps.tsx]: Why DialogLoaderGoogleAuthStepsContainer instead DialogLoaderGoogleAuthSteps?

File: packages/shared/modules/wallet-transfer/components/wallet-transfer-form.tsx [hunk 14]
```diff
@@ -3,7 +3,7 @@ import "./wallet-transfer-form.scss";
 import { FormikProps, withFormik } from "formik";
 import { WalletData } from "gv-api-web";
 import { GVButton, GVFormikField, GVTextField } from "gv-react-components";
-import * as React from "react";
+import React, { ComponentType } from "react";
```
Comment (pavelzubov) [inline packages/shared/modules/wallet-transfer/components/wallet-transfer-form.tsx]: Why?

**Comment by Anchello** [comment]
Fixed all comments

**Pull request merged**